src/frontend/src/binder/values.rs

```diff
@@ -80,6 +81,25 @@ impl Binder {
             schema,
         })
     }
+
+    /// Bind row to `struct_value` for nested column,
+    /// e.g. Row(1,2,(1,2,3)).
+    /// Only accept value and row expr in row.
+    pub fn bind_row(&mut self, exprs: &[Expr]) -> Result<StructValue> {
```

Comment:
```suggestion
    pub fn bind_row(&mut self, exprs: &[Expr]) -> Result<Literal> {
```

pls include the wrapping of StructValue into Literal here.

```
                let value = self.bind_row(&exprs)?;
                let data_type = DataType::Struct {
                    fields: value
                        .get_fields()
                        .iter()
                        .map(DataType::datum_type_to_data_type)
                        .collect::<Result<Vec<_>>>()?
                        .into(),
                };
                Ok(ExprImpl::Literal(Box::new(Literal::new(
                    Some(value.to_scalar_value()),
                    data_type,
                ))))
```